### DISCUSSION TITLE
v2.6(a)

### DISCUSSION THREAD

**Discussion by Alfredredbird** [Alfredredbird/tookie-osint#53] [General]
This release includes the command line version of Tookie as well as the WebUI version

<hr /><em>This discussion was created from the release <a href='https://github.com/Alfredredbird/tookie-osint/releases/tag/v2.5(a)'>v2.6(a)</a>.</em>

**Comment by TermuxHackz** [comment]
Hello @Alfredredbird , I’m the owner of the repo TermuxHackz and tool [X-Osint](https://github.com/TermuxHackz/X-osint), writing this to seek permission to integrate your tool with mine, necessary credits would be given and source, Looking forward to hearing from you

**Reply by Alfredredbird** [reply]
Hey, I'm a big fan of your project, would you be able to message me privately on Discord so we can talk more about this? I'm not a fan of making deals on GitHub. 

Discord: alfredredbird2

**Comment by TermuxHackz** [comment]
@Alfredredbird Sent a request..
